client/src/index.jsx

```diff
@@ -1,15 +1,37 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import $ from 'jquery';
+import Bonuses from './components/bonuses.jsx';
+
 
 class App extends React.Component {
   constructor(props) {
     super(props);
+    this.state = {
+      bundleNumber: 1, //number
+      bonus: null, //object
+      selectedSong: '', //url
```

Comment:
When your other two commented out indicators tell you the data type it's expecting, `//url` probably should be something like, `//string, representing url`.

Reply:
okay